adr/ADR-1.md

```diff
@@ -97,6 +97,28 @@ Today the list API's just return `["ORDERS"]`, these will become:
 
 With the same `error` treatment when some error happens.
 
+### Numerical Values
+
+Some fields commuincated in the JetStream API are unsigned 64 bit values (uint64). These will be communicated as numbers in json. Clients must handle them, even if it means custom parsing. Commonly fields that represent a message sequence or id, or a field that holds a duration of nanoseconds are uint64 values. The most up-to-date information on specific fields can be found in the [schema](https://github.com/nats-io/jsm.go/tree/main/schemas/jetstream/api/v1).
```

Comment:
lets also call out some are specifically 32bit and clients should be careful not to overflow them